### ISSUE TITLE
Add learning rate to the metrix

### ISSUE THREAD

**Issue by JM-IP** [larq/larq#177]
The birealnet use linear_cosine as the learning schedule.
How can I print out the learning rate after each epoch?

**Comment by lgeiger** [comment]
Learning rate schedules defined using `tf.keras.callbacks.LearningRateSchedule` will show up in TensorBoard automatically. I am not sure about `tf.keras.optimizers.schedules`, since it is a quite new API.

This question is related to TensorFlow and better asked at https://github.com/tensorflow/tensorflow/issues or https://stackoverflow.com/questions/tagged/tensorflow.
